data/sql/updates/pending_db_world/rev_1680983241816980712.sql

```diff
@@ -0,0 +1,134 @@
+-- Ancestral Spirit Wolf SAI (Source: Sniffs & https://www.youtube.com/watch?v=xFF8DOSeMcg)
+SET @ID := 17077;
+UPDATE `creature_template` SET `speed_walk` = 1.6, `speed_run` = 1, `unit_flags` = 768, `AIName` = "SmartAI", `ScriptName` = "" WHERE `entry` = @ID;
```

Comment:
Speeds are already sniffed: https://github.com/neifion-00000000/azerothcore-wotlk/blob/76e9f511a1d3f08f3fe9d224c7ae488c39930a5c/data/sql/updates/pending_db_world/rev_1672823650132911600.sql
```suggestion
UPDATE `creature_template` SET `unit_flags` = 768, `AIName` = "SmartAI", `ScriptName` = "" WHERE `entry` = @ID;
```

Reply:
Updated accordingly, but still facing the issue where the script is behaving inconsistently. Can we add a WIP label to the PR? I don't know how or don't have permission to do so.